app/src/main/java/com/example/ohhi_android/SignUpScreen.java

```diff
@@ -0,0 +1,139 @@
+package com.example.ohhi_android;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SignUpScreen extends AppCompatActivity {
+    Animation anim1, anim2, anim3;
+    ImageView Logo;
+    TextView head, desc;
+    Button getStarted;
+    ImageView picAdd;
+    final static private int PICK_IMAGE = 1;
+    FirebaseAuth mAuth;
+    FirebaseFirestore mstore;
```

Comment:
```suggestion
    FirebaseFirestore mStore;
```

Consistency with naming.